index.html

```diff
@@ -4,7 +4,64 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
- [ ] Design-wise your page looks awesome. But I noticed that you used fixed width to reproduce mobile width, but this is a wrong approach because if you open it in mobile phones the page will be broken. Please use google developer tools to check the content layout in mobile view:

https://user-images.githubusercontent.com/77038610/158801329-8e77ebb0-0a09-4dd9-aca5-144bc83ddcda.mp4